hide_invoice_shipping_address/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Roger Sans <roger.sans@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Hide invoice shipping address',
+    'version': '14.0.1.0',
+    'author': 'Sygel',
```

Comment:
You should include Odoo Community Association (OCA) as an author

Reply:
Done